.pre-commit-config.yaml

```diff
@@ -19,3 +14,13 @@ repos:
       - id: end-of-file-fixer
       - id: no-commit-to-branch
         args: [--branch, develop, --branch, master, --pattern, release/.*]
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
```

Comment:
I think you can use black 22.3.0 if you're using an isolated python 3.7 to run pre-commit. https://github.com/NCI-GDC/cookiecutter-pystones/blob/8727386f7d9b5554b8f7d2a6fd7e10a640465943/%7B%7B%20cookiecutter.package_name%20%7D%7D/.pre-commit-config.yaml#L27

Reply:
I think 19.10b0 is the version that supports our current Jenkins pipeline. 
I will upgrade black and remove Jenkins.